config/discovery.go

```diff
@@ -42,6 +42,9 @@ type Discovery struct {
 	DeactivateAfter Duration
 	// PollOverrides configures polling for specific providers.
 	PollOverrides []Polling
+	// ProviderReloadInterval is the amount of time to wait between reloading
+	// providers from the datastore. Set to 0 to disable periodic reloading.
+	ProviderReloadInterval Duration
```

Comment:
Might be worth mentioning that this is only required when find/ingest are running as seperate processes where the find process does not have access to the up to date information that the ingest process is writing.